### DISCUSSION TITLE
Winforms - Browser Hang/Not responding when load large html table

### DISCUSSION THREAD

**Discussion by laxmi-lal-menaria** [cefsharp/CefSharp#4631] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

117.2.40.0

### Operating System

Windows Server 2016

### Architecture

x64

### .Net Version

.Net 4.8.2

### Implementation

WinForms

### Reproduction Steps

Have load a  large page with html table which has more than 5000 rows, unable to scroll to the end of the page.

### Expected behavior

Should work like normal browser

### Actual behavior

Browser hang

### Does this problem also occur in the CEF Sample Application

Not Tested

**Comment by laxmi-lal-menaria** [comment]
any suggestion?

**Comment by amaitland** [comment]
> ### CefSharp Version
> 117.2.40.0

Have you tested with the latest version? M119 at time of writing

How does `Chrome` behave loading the same html?

Does the problem reproduce using the [MinimalExample](https://github.com/cefsharp/CefSharp.MinimalExample)?

> Does the cef log provide any relevant information? (By default there should be a debug.log file in your bin directory)

Have you checked the log file for errors?

**Comment by laxmi-lal-menaria** [comment]
- No errors at console, tried devtools too.
- Chrome is working fine when I load the same page with same data.
- Tied with latest stable version.
- Will try with Minimal Example

**Reply by amaitland** [reply]
> * No errors at console, tried devtools too.

Check the log **file**. See https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file for additional details

**Comment by laxmi-lal-menaria** [comment]
Noting in logs related to it.


Let me explain my issue:

I have a page, after load (!e.IsLoading) I am filling the form, after fill, clicking on the button using javascript call. 
These are working fine.
Once the javascript AJAX completed, getting json data and populating the HTML table. If this html table is low volume then its working fine.

But When have Million rows, unable to do any javascript click.. 

I have done these things  using Timer, ie checking the HTML table filled or not using timer tick event. I think Time is issue. 

Need to look who can I detect that html table is rendered without using timer?

**Comment by laxmi-lal-menaria** [comment]
Only this :
[1127/044144.737:WARNING:policy_logger.cc(151)] :components\enterprise\browser\controller\chrome_browser_cloud_management_controller.cc(88) Could not create policy manager as CBCM is not enabled.
[1127/044145.030:WARNING:angle_platform_impl.cc(49)] RendererVk.cpp:1962 (initialize): ANGLE VMA version: 2003000
